src/kafka.c

```diff
@@ -54,6 +54,78 @@ rebalance_cb (rd_kafka_t *rk, rd_kafka_resp_err_t err, rd_kafka_topic_partition_
     }
 }
 
+/*
+ * replace_char
+ *      Replace in string `str` character c with character s.
+ */
+static void replace_char(char *str, char c, char s)
```

Comment:
`utils/helper.c` could be a better fit for the location of this function.

Reply:
it is the only module where this function is used, not sure if we want to extract it on the higher level.